src/components/Users/UsersContainer.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import Users from "./Users";
+import { connect } from "react-redux";
```

Comment:
This is a third-party library import - move it to the appropriate group.

Reply:
done